index.js

```diff
@@ -1,7 +1,8 @@
-module.exports = function (main) {
-  main = main || require.main
-  if (isIISNode(main)) return handleIISNode(main)
-  else return main.filename
+module.exports = function (_require) {
+  _require = _require || require
+  var main = _require.main
+  if (main && isIISNode(main)) return handleIISNode(main)
+  else return main ? main.filename : process.cwd()
```

Comment:
`else` not technically needed here

Also, I know it probably doesn't matter, but `process.cwd()` returns a directory path instead of a file path. I guess in the edge-case of a webpack context, it won't be used anyway so shouldn't matter?